### PR TITLE
Add list of required packages.

### PR DESCRIPTION
**Pull request by kortschak**
@cznic Please take a look.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -6,4 +6,14 @@ Installation
 
     $ go get github.com/cznic/crt
 
+Installation requires the following Go packages
```
Comment (cznic) [inline README.md]: AFAICT, the installation, like in `go get` and `go install` does not require these tools. They're just called by the Makefile, which is supposed to be used only by a developer. Of course, documenting the list needed for running `make` is a good idea, please just adjust the meaning of the above sentence, thanks.

**Review by cznic — CHANGES_REQUESTED** [review]
Please address the inline comment, thank you.

Afterwards it's LGTM and please commit to the master branch, thanks.

**Comment by kortschak** [comment]
PTAL

BTW Do you prefer linear or branched (with merge commits) history?

**Review by cznic — APPROVED** [review]
LGTM, thanks.

**Comment by cznic** [comment]
> BTW Do you prefer linear or branched (with merge commits) history?

I like `git merge --squash some-branch`, but feel free to do what you prefer.

**Pull request merged**